addons/website_sale/views/templates.xml

```diff
@@ -882,6 +868,48 @@
         </div>
     </template>
 
+    <template
+        id="website_sale.products_categories_list_collapsible"
+        inherit_id="website_sale.products_categories_list"
+        name="Collapsed eCommerce Categories"
+    >
+        <xpath expr="." position="replace">
+            <div t-attf-class="{{not isOffcanvas and 'accordion-item'}}">
+                <h6 t-attf-class="accordion-header {{_titleClasses}}">
+                    <button
+                        class="accordion-button px-0 bg-transparent shadow-none"
```

Comment:
I find the arrows a bit too big, no? I propose
```css
--accordion-btn-icon-width: 1rem;
```

| this PR | proposal |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/96a650e7-b3dc-4318-a5c9-5df82f247b12) | ![image](https://github.com/user-attachments/assets/a7313a9f-a1c8-46ef-aae3-68651f72e7b0) |